dev-packages.nix

```diff
@@ -203,5 +203,6 @@ in
   # which sets LD_LIBRARY_PATH and others correctly.
   cabal-wrapper
```

Comment:
Do we still need `cabal-wrapper` after this, and all the `compile-deps` in there? I'd assume it'd become less, as we only need to compile `wire-server` itself (and can use the `shellFor` stuff for that?)

Reply:
We don't, its gone.